specs/recipient_spec.rb

```diff
@@ -0,0 +1,72 @@
+require_relative "test_helper"
+# THIS IS MY COMMENT - Jessica
+
+describe "Recipient" do
+  let(:slack_id) { 20 }
+  let(:username) { "Sopheary" }
+  let(:recipient) { Slack::Recipient.new(slack_id, username) }
+
+  describe "Initialization" do
+    it "can be instantiated" do
+      expect(recipient).must_be_instance_of Slack::Recipient
+      expect(recipient.slack_id).must_equal 20
+      expect(recipient.name).must_equal "Sopheary"
+    end
+  end
+
+  describe "send message" do
+    it "sends a valid message" do
+      VCR.use_cassette("slack_message") do
+        users = Slack::User.list
+        return_value = users[0].send_message("hi")
+        expect(return_value).must_equal true
+      end
+    end
+
+    it "raises an error if api sends back not okay" do
```

Comment:
You should test this with both an invalid token and again with an empty message.